tardis/apps/publication_workflow/static/js/lib/angular-material-data-table-0.10.10/dist/md-data-table.js

```diff
@@ -0,0 +1,1486 @@
+    } else {
+      return $controller(options.controller, inject);
+    }
+  }
+  
+  function getTemplate(options) {
+    return $q(function (resolve, reject) {
+      var template = options.template;
+      
+      function illegalType(type) {
+        reject('Unexpected template value. Expected a string; received a ' + type + '.');
+      }
+      
+      if(template) {
+        return angular.isString(template) ? resolve(template) : illegalType(typeof template);
+      }
+      
+      if(options.templateUrl) {
+        template = $templateCache.get(options.templateUrl);
+        
+        if(template) {
+          return resolve(template);
+        }
+        
+        var success = function (template) {
+          return resolve(template);
+        };
+        
+        var error = function () {
+          return reject('Error retrieving template from URL.');
+        };
+        
+        return $templateRequest(options.templateUrl).then(success, error);
+      }
+      
+      reject('Template not provided.');
+    });
+  }
+  
+  function logError(error) {
+    busy = false;
+    console.error(error);
+  }
+  
+  function escToClose(element) {
+    var keyup = function (event) {
+      if(event.keyCode === ESCAPE) {
+        element.remove();
+      }
+    };
+    
+    body.on('keyup', keyup);
+    
+    element.on('$destroy', function () {
+      body.off('keyup', keyup);
+    });
+  }
+
+  function focusOnOpen(element) {
+    $mdUtil.nextTick(function () {
+      var autofocus = $mdUtil.findFocusTarget(element);
+      
+      if(autofocus) {
+        autofocus.focus();
+      }
+    }, false);
+  }
+
+  function positionDialog(element, target) {
+    var table = angular.element(target).controller('mdCell').getTable();
+    
+    var getHeight = function () {
+      return element.prop('clientHeight');
+    };
+    
+    var getSize = function () {
+      return {
+        width: getWidth(),
+        height: getHeight()
+      };
+    };
+    
+    var getTableBounds = function () {
+      var parent = table.parent();
+      
+      if(parent.prop('tagName') === 'MD-TABLE-CONTAINER') {
+        return parent[0].getBoundingClientRect();
+      } else {
+        return table[0].getBoundingClientRect();
+      }
+    };
+    
+    var getWidth = function () {
+      return element.prop('clientWidth');
+    };
+    
+    var reposition = function () {
+      var size = getSize();
+      var cellBounds = target.getBoundingClientRect();
+      var tableBounds = getTableBounds();
+      
+      if(size.width > tableBounds.right - cellBounds.left) {
+        element.css('left', tableBounds.right - size.width + 'px');
+      } else {
+        element.css('left', cellBounds.left + 'px');
+      }
+      
+      if(size.height > tableBounds.bottom - cellBounds.top) {
+        element.css('top', tableBounds.bottom - size.height + 'px');
+      } else {
+        element.css('top', cellBounds.top + 1 + 'px');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933951)
```suggestion
        element.css("top", cellBounds.top + 1 + 'px');
```